### ISSUE TITLE
Tabs: Screen readers cannot read text inside tab panels

### ISSUE THREAD

**Issue by NoahGresser** [wet-boew/wet-boew#8922]
Similar to issue #8918, neither NVDA on Chrome 84.0.4147.125 nor NVDA on Firefox 79 (I do not have JAWS to test), can read the text inside the expanded tab panel in [the example for the tabbed interface](https://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html). Neither ARROW keys nor TAB allows the screen reader to access the text inside the panel when the panel does not have interactive elements like form fields or links that can be TABed to.

In comparison, this [implementation of a tablist by the W3C](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html) makes the tab panel itself focusable, so it can be read by screen readers. However, this may not be feasible with WET's current implementation of the tabbed interface, as in WET's implementation the tab panels are not adjacent to their respective tabs in the DOM.

**Comment by Ricokola** [comment]
After a brief research on this issue I can confirm that the WET-BOEW Tabbed Interface plugin interaction behaves exactly as prescribed by W3C for tabs keyboard interaction:  [https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/).  When focus is on a tab in a tablist with either horizontal or vertical orientation: “Space” or “Enter” activates the tab if it was not activated automatically on focus.
